backend/routes/ClassData.go

```diff
@@ -0,0 +1,140 @@
+// Package routes includes the route handlers for sd schedule planner
+package routes
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/ucsdscheduleplanner/UCSD-Schedule-Planner/backend/db"
+)
+
+const logTagClassData = "[ClassData]"
```

Comment:
I have no idea what this variable does, is it like a log id?

Reply:
With this, the logs will look like:

```
[ClassData] Http Error, not found...
[Type] Http Error, not found...
...
```

Reply:
Maybe there is better way to do it but this is what I learned from school :(

Reply:
when I say I don't know what a variable does, it means it is not immediately clear when reading the variable what purpose it serves.

You should have another file with a list of consts that have intuitive ids. Remember part of how you program is so that future devs can understand what you write.